cognite/client/data_classes/capabilities.py

```diff
@@ -98,9 +98,6 @@ def as_tuples(self) -> set[tuple]:
             # Basic implementation for all simple Scopes (e.g. all or currentuser)
             return {(self._scope_name,)}
 
-        def is_within(self, other: Self) -> bool:
```

Comment:
Surprised that these slipped through the reviews.

Reply:
I actually kept it as a "simple 1:1 check", but decided now with the raw-database changes to not duplicate the error-prone scope logic.